packages/fixie/package.json

```diff
@@ -22,6 +22,7 @@
   "bin": "./dist/main.js",
   "dependencies": {
     "@apollo/client": "^3.8.1",
+    "@fixieai/sdk": "*",
```

Comment:
Can this dependency be delegated to the agent project? I'm starting to think it needs to be a dependency of the agent project anyway for the peerDependency resolution to work properly with ai-jsx.

Reply:
I don't know. I was unable to get this to work without putting this in here. It might have to do with the fact that I'm running it locally using yarn.

Reply:
Yeah, that works. Done.